openpype/hosts/houdini/plugins/publish/collect_mantra_rop.py

```diff
@@ -31,49 +31,63 @@ class CollectMantraROPRenderProducts(pyblish.api.InstancePlugin):
     def process(self, instance):
 
         rop = hou.node(instance.data.get("instance_node"))
+        frame_start = instance.data["frameStart"]
+        frame_end = instance.data["frameEnd"]
```

Comment:
```suggestion
        frame_start = str(instance.data["frameStart"])
        frame_end = str(instance.data["frameEnd"])
```
If it is not a string it will error out in the collectors. See the log below.
```
DEBUG: Chunk Size: 10
Traceback (most recent call last):
  File "D:\kayla\OpenPype\.venv\lib\site-packages\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "D:\kayla\OpenPype\openpype\hosts\houdini\plugins\publish\collect_mantra_rop.py", line 54, in process
    "beauty": file_collection.generate_expected_filepaths(
  File "D:\kayla\OpenPype\openpype\pipeline\file_collection.py", line 79, in generate_expected_filepaths
    files = collect_filepaths_from_sequential_path(
  File "D:\kayla\OpenPype\openpype\pipeline\file_collection.py", line 30, in collect_filepaths_from_sequential_path
    os.path.dirname(file_path))
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.435/python39\lib\ntpath.py", line 223, in dirname
    return split(p)[0]
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.435/python39\lib\ntpath.py", line 185, in split
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not float
```

Reply:
It doesn't look the frame start and end values are causing this issue. Also, those values are usually integer numbers. Please look carefully into the trackback.

Reply:
it seems like the issue is in the order of input attributes. Fixing it

Reply:
The problem resolved after the change of the order of input attributes